src/zenml/cli/stack.py

```diff
@@ -2430,25 +2496,31 @@ def _get_stack_component_info(
             selected_orchestrator_idx
         ]
 
+        config = {}
         if selected_orchestrator[0] == "Sagemaker":
             flavor = "sagemaker"
-            execution_role = Prompt.ask("Please enter an execution role ARN:")
-            config = {"execution_role": execution_role}
+            execution_role = Prompt.ask("Enter an execution role ARN:")
+            config["execution_role"] = execution_role
         elif selected_orchestrator[0] == "Skypilot (EC2)":
             flavor = "vm_aws"
-            config = {"region": selected_orchestrator[1]}
+            config["region"] = selected_orchestrator[1]
+        elif selected_orchestrator[0] == "Skypilot (Compute)":
+            flavor = "vm_gcp"
+            config["region"] = query_gcp_region()
```

Comment:
Agreed, I will wait for your PR  and extend `GCPZenMLCloudStackDeployment` with relevant for Vertex info.

Reply:
Alright, but I don't recommend extending anything. Just give them all the regions there are and let them choose one that has Vertex enabled. I would like to avoid us having to maintain the list of Vertex enabled GCP regions.

Reply:
addressed